python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-bgp.j2

```diff
@@ -69,6 +68,16 @@ ASN Notation: {{ router_bgp.as_notation | arista.avd.default('asplain') }}
 {%             elif router_bgp.bgp.additional_paths.send_limit is arista.avd.defined and router_bgp.bgp.additional_paths.send == 'limit' %}
 | bgp additional-paths send limit {{ router_bgp.bgp.additional_paths.send_limit }} |
 {%             endif %}
+{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.any is arista.avd.defined(true) %}
+| bgp additional-paths send any |
+{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.backup is arista.avd.defined(true) %}
+| bgp additional-paths send backup |
+{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp is arista.avd.defined(true) %}
+| bgp additional-paths send ecmp |
+{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp_limit is arista.avd.defined %}
+| bgp additional-paths send ecmp limit {{ router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp_limit }} |
+{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.limit is arista.avd.defined %}
+| bgp additional-paths send limit {{ router_bgp.address_family_evpn.bgp_additional_paths.send.limit }} |
```

Comment:
all these are EVPN address family related - would be better in another table

Reply:
plus today we are not even documenting the non deprecated keys `router_bgp.address_family_evpn.bgp.additional_paths.<FOO>` (with a dot after bgp) so not sure you need to add this really.